src/typings/gravity.d.ts

```diff
@@ -1,14 +1,17 @@
-interface User {
-  roles?: UserRole[]
-  accessToken?: string
-  appToken?: string
-  email?: string
-  has_partner_access?: string
-  id?: string
-  lab_features?: string[]
-  type?: string
-  name?: string
-}
+type User =
+  | {
+      roles?: UserRole[]
+      accessToken?: string
+      appToken?: string
+      email?: string
+      has_partner_access?: string
+      id?: string
```

Comment:
If the User is exist-y; then I think they are _always_ going to have an `id`

Reply:
Ah shoot, that blew up all over 😬 gonna just keep it how it was for now